extensions/ql-vscode/src/view/common/Alert.tsx

```diff
@@ -0,0 +1,98 @@
+import * as React from 'react';
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+type ContainerProps = {
+  type: 'warning' | 'error';
+  inverse?: boolean;
+};
+
+const getBackgroundColor = ({ type, inverse }: ContainerProps): string => {
+  if (!inverse) {
+    return 'var(--vscode-notifications-background)';
+  }
+
+  switch (type) {
+    case 'warning':
+      return 'var(--vscode-editorWarning-foreground)';
+    case 'error':
+      return 'var(--vscode-debugExceptionWidget-border)';
+  }
+};
+
+const getTextColor = ({ type, inverse }: ContainerProps): string => {
+  if (!inverse) {
+    return 'var(--vscode-tab-activeForeground)';
+  }
+
+  switch (type) {
+    case 'warning':
+      return 'var(--vscode-panel-background)';
+    case 'error':
+      return 'var(--vscode-list-activeSelectionForeground)';
```

Comment:
```suggestion
      return 'var(--vscode-editor-foreground)';
```
Changes from #ffffff to #d4d4d4

Reply:
Unfortunately, this makes the text quite unreadable when combined with the different error background color:

![Screenshot 2022-09-26 at 13 56 11](https://user-images.githubusercontent.com/1112623/192270194-c2bbb13a-d694-45b5-88df-49bcb019413f.png)

Even with the current background color, this makes it harder to read the text.